ur_simulation_ignition/launch/ur_sim_control.launch.py

```diff
@@ -186,6 +178,17 @@ def launch_setup(context, *args, **kwargs):
         ],
     )
 
+    # ros_ign_bridge (clock -> ROS 2)
```

Comment:
Why is this needed?

Reply:
Otherwise the /clock topic isn't populated which is used e.g. by the robot_state_publisher (see attached screenshot from rqt_graph) resulting in an error message such as this one: 
`[rviz2-9] [WARN] [1644917311.649767236] [moveit_ros.planning_scene_monitor.planning_scene_monitor]: Maybe failed to update robot state, time diff: 1644917299.652s`

![rqt_graph_clock_cropped](https://user-images.githubusercontent.com/33007068/154042907-d92ff5be-b5b8-4b69-8f24-4323ab5874fe.png)